steps/generate_analytical_dataset.py

```diff
@@ -8,14 +8,19 @@
 import re
 import os
 import concurrent.futures
+import time
+import datetime
 
 from Crypto.Cipher import AES
 from Crypto.Util import Counter
 from pyspark.sql.types import *
 from pyspark.sql import Row
 from datetime import datetime
-import logging
 import pytz
+from logger import setup_logging
+
+
+the_logger = setup_logging("DEBUG", log_path = "/var/log/adg/generate_analytical_dataset_log")
```

Comment:
log levels should come from the environment, not hard coded. Is there a reason for doing this?